code/modules/mob/living/carbon/carbon.dm

```diff
@@ -249,7 +249,12 @@
 			// BEGIN HUGCODE - N3X
 			else
 				playsound(get_turf(src), 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
```

Comment:
is there another sound to use?

Reply:
I dunno... I figured if it's good enough for headpats it's probably good enough for nose boops? But there is a nose_boop.ogg on another server, maybe I steal that?

Reply:
Double check the licensure on the server to see if you can "steal" it ;)

Reply:
I haven't taken anything yet, maintainers wished to primarily use already existing sounds if possible, so I did :)